examples/user_interfaces/embedding_in_wx3.py

```diff
@@ -134,19 +137,22 @@ def OnInit(self):
         plot_container.SetSizer(sizer)
 
         # whiz button ------------------
-
         whiz_button = xrc.XRCCTRL(self.frame, "whiz_button")
-        wx.EVT_BUTTON(whiz_button, whiz_button.GetId(),
-                      self.plotpanel.OnWhiz)
+        if 'phoenix' in wx.PlatformInfo:
+            whiz_button.Bind(wx.EVT_BUTTON, self.plotpanel.OnWhiz)
+        else:
+            wx.EVT_BUTTON(whiz_button, whiz_button.GetId(),
+                          self.plotpanel.OnWhiz)
```

Comment:
with the little bit of wx that I learned to write up my book, shouldn't we be able to do the Bind() approach in the versions of wxWidgets that we support? If so, then I would just get rid of the old style and use Bind() instead.